frontend/src/static/js/components/webstatus-overview-cells.ts

```diff
@@ -370,59 +382,103 @@ export const CELL_DEFS: Record<ColumnKey, ColumnDefinition> = {
       ],
     },
   },
+  [ColumnKey.DesktopAvailabilityChrome]: {
+    nameInDialog: 'Availibility in desktop Chrome',
+    group: 'Availability',
+    headerHtml: html``,
+    unsortable: true,
+    cellClass: 'centered',
+    cellRenderer: renderDesktopAvailablity,
+    options: {browser: 'chrome', channel: 'stable'},
```

Comment:
renderDesktopAvailablity doesn't need channel
```suggestion
    options: {browser: 'chrome'},
```